conf/modules.config

```diff
@@ -0,0 +1,135 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Config file for defining DSL2 per module options and publishing paths
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Available keys to override module options:
+        ext.args   = Additional arguments appended to command in module.
+        ext.args2  = Second set of arguments appended to command in module (multi-tool modules).
+        ext.args3  = Third set of arguments appended to command in module (multi-tool modules).
+        ext.prefix = File name prefix for output files.
+----------------------------------------------------------------------------------------
+*/
+
+process {
+
+    publishDir = [
+        path: { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" },
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+    ]
+
+    withName: FASTQC {
+        ext.args = '--quiet'
+    }
+
+    withName: 'MULTIQC' {
+        ext.args   = { params.multiqc_title ? "--title \"$params.multiqc_title\"" : '' }
+        publishDir = [
+            path: { "${params.outdir}/multiqc" },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+    }
+
+    withName: FASTQC {
+        ext.args = '--quiet'
+    }
+
+    withName: KRAKEN2_KRAKEN2 {
+        ext.args = '--report-minimizer-data'
+    }
+
+    withName: KRAKEN_MERGE {
+        ext.prefix = "${params.file_prefix}"
+    }
+
+    withName: SOURCEPREDICT {
+        ext.prefix = "${params.file_prefix}"
+    }
+
+    withName: SAM2LCA_MERGE {
+        ext.prefix = "${params.file_prefix}"
+    }
+
+    withName: PYDAMAGE_MERGE {
+        ext.prefix = "${params.file_prefix}"
+    }
+
+    withName: DAMAGEPROFILER_MERGE {
+        ext.prefix = "${params.file_prefix}"
+    }
+
+    withName: PYDAMAGE_ANALYZE {
+        ext.prefix = {"${meta.id}-${meta.genome_name}"}
+        ext.args = [
+            '--group'
+        ].join(' ').trim()
+    }
+
+    withName: SAM2LCA_PREPDB {
+        publishDir = [
+            enabled: false
+        ]
+    }
+
+    withName: SAM2LCA_ANALYZE {
+        ext.args = [
+            "--identity ${params.sam2lca_identity}",
+            "--acc2tax ${params.sam2lca_acc2tax}"
+        ].join(' ').trim()
+
+        publishDir = [
+            path: { "${params.outdir}/sam2lca" },
+            mode: params.publish_dir_mode,
+            pattern: '*.{csv,json}'
+        ]
+    }
+
+    withName: QUARTONOTEBOOK {
+        ext.prefix = "${params.file_prefix}_report"
+        singularity.registry  = 'cloud.sylabs.io'
+        apptainer.registry    = 'cloud.sylabs.io'
```

Comment:
What's the point of having these registries here? The module seems to use Seqera containers :)